src/createStore.js

```diff
@@ -22,8 +24,8 @@ export default ({
     initialState,
     compose(
       applyMiddleware(thunk, ...middleware),
-      ...enhancers,
-      devtools
+      devtools,
+      ...enhancers
```

Comment:
This might get weird. For example, an enhancer like `persistState()` from DevTools package should be applied after DevTools because it persists *its* state. However, it would be surprising for the user if an enhancer like `redux-localstorage` suddenly started operating on DevTools state. Such enhancers should generally be *before* DevTools.